tests/Driver/PgSQL/DriverTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Driver\PgSQL;
+
+use Doctrine\DBAL\Driver as DriverInterface;
+use Doctrine\DBAL\Driver\PgSQL\Driver;
+use Doctrine\DBAL\Tests\Driver\AbstractPostgreSQLDriverTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+
+class DriverTest extends AbstractPostgreSQLDriverTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        if (isset($GLOBALS['db_driver']) && $GLOBALS['db_driver'] === 'pgsql') {
+            return;
+        }
+
+        self::markTestSkipped('Test enabled only when using pgsql specific phpunit.xml');
+    }
+
+    /**
+     * Ensure we can handle URI notation for IPv6 addresses
+     */
+    public function testConnectionIPv6(): void
+    {
+        self::expectNotToPerformAssertions();
+
+        $params         = TestUtil::getConnectionParams();
+        $params['host'] = '[::1]';
+
+        $this->driver->connect($params);
+    }
```

Comment:
This test only works if there's a Postgres server listening on localhost. That's the case in our CI, but we cannot assume that every contributor runs that setup.

Can you maybe check if the host is currently set to `localhost` or `127.0.0.1` and skip the test otherwise?

Reply:
This is resolved now, should I change the PR target or is 3.9 still valid?
And sorry for the delay!